src/Modal/Modal.jsx

```diff
@@ -13,12 +23,29 @@ class Modal extends Component {
   }
 
   render() {
-    const { onClose, children } = this.props;
+    const { onClose, role, ariaLabel, children } = this.props;
     return (
-      <ModalStyles>
-        <Overlay onClick={onClose} />
-        <Content>{children}</Content>
-      </ModalStyles>
+      <ReactFocusTrap
+        tag="aside"
+        focusTrapOptions={{ onDeactivate: onClose }}
+        aria-modal="true"
+        role={role}
+        aria-label={ariaLabel}
+      >
+        <ModalStyles>
+          <Overlay onClick={onClose} />
+          <Content>
+            <Row textAlign="right">
+              <Col offset={11} lg={1}>
+                <CloseButton ref={this.onClose} onClick={onClose}>
+                  <Icon name="close" />
+                </CloseButton>
+              </Col>
+            </Row>
+            <div style={{ marginRight: 24 }}>{children}</div>
```

Comment:
I thought we will keep all the styling in the styles files? 

Also the left margin should be the same: 
![image](https://user-images.githubusercontent.com/492636/56203379-63823b80-6045-11e9-9d98-6ab3899ce288.png)